### ISSUE TITLE
Why do you freeze batch norm layer for fine-tuning?

### ISSUE THREAD

**Issue by Seyoung9304** [princeton-vl/RAFT#141]
Hi, this is a very nice work! Thanks for your contribution.

In your code, I found that you [freeze every batch normalization layer](https://github.com/princeton-vl/RAFT/blob/224320502d66c356d88e6c712f38129e60661e80/train.py#L147) when you finetune the model.  
```
if args.stage != 'chairs':
        model.module.freeze_bn()
```

I wonder why you froze BN layer when you fine tune the model. Is there any theoretical or experimental reason for it?

Thank you in advance :)
